modules/dnn/test/test_layers_1d.cpp

```diff
@@ -42,8 +42,8 @@ TEST_P(Layer_1d_Test, Scale)
 
     cv::Mat output_ref = input.mul(weight);
     runLayer(layer, inputs, outputs);
-
-    ASSERT_EQ(shape(output_ref), shape(outputs[0]));
+    ASSERT_EQ(1, outputs.size());
+    ASSERT_EQ(shape(outputs[0]), shape(output_ref));
```

Comment:
Reference should be put in the first parameter and real output goes to the second.

See https://github.com/opencv/opencv/pull/25231#discussion_r1555470965.

Reply:
Done